.github/workflows/test.yml

```diff
@@ -0,0 +1,73 @@
+name: Test
+
+on:
+  pull_request:
+    branches:
+      - main
+
+concurrency: ${{ github.workflow }}
+
+jobs:
+  test:
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    runs-on: macos-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+
+      - name: Setup bun
+        uses: oven-sh/setup-bun@v2
+
+      - name: Setup Xcode
+        uses: maxim-lobanov/setup-xcode@v1
+        with:
+          xcode-version: ~16.2
+
+      - name: Configure Git user
+        run: |
+          git config --global user.name "Candle Finance CI"
+          git config --global user.email "contact@trycandle.com"
+
+      - name: Configure npm auth
+        run: echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
+
+      - name: Install dependencies
+        run: bun install
+
+      - name: Install swift-format
+        run: brew install swift-format
```

Comment:
#q why install swift-format if you're not going to run it?

Reply:
It's being run in the specs command.